css/style.css

```diff
@@ -780,18 +1938,123 @@
     flex-direction: column;
     padding: 11px 0 55px 48px;
   }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)